common/app/views/fragments/page/head/stylesheets/styles.scala.html

```diff
@@ -60,9 +60,5 @@
 @styles.criticalCss
 @styles.linkCss
 <!--<![endif]-->
-<style class="js-loggable">
```

Comment:
Can't the ad free CSS stay? Still seems a bit better than using JS

Reply:
If we're going to import and use remove-slots anyway then I'm not sure if there's value in doing the same job twice?